docs/UserGuide.md

```diff
@@ -685,25 +758,63 @@ folder circled in red.
 
 ![stockCsvExample1](images/stockCsvExample1.png)
  
-Open `stocks.csv` folder circled in red with excel.
+Open `stocks.CSV` folder circled in red with excel.
 
 ![stockCsvExample2](images/stockCsvExample2.png)
 
 The stocks in your inventory will be listed according the to format shown below. The file also includes
-the latest update time for the `stocks.csv`.
+the latest update time for the `stocks.CSV`.
 
 ![stockCsvExample3](images/stockCsvExample3.png)
 
+### Clearing all data in Warenager: `clear`
+Clears all the existing data in Warenager.
+
+<h4>Format</h4>
+
+```
+clear
+```
+<div markdown="block" class="alert alert-warning" markdown="1">
+
+**:warning: Warning for clearing data**<br>
+`clear` is an irreversible process. Do backup the data it if might be needed again.
+
+</div>
+
+<h4>Below is the expected outcome for clearing Warenager:</h4>
+
+**Before input**:
+
+![clear_step1](images/clear/clear_step1.png)
+
+
+**After input**:
+
+![clear_step2](images/clear/clear_step2.png)
+
+
+
+### Toggling tabs: `tab`
+Toggles between the tabs in Warenager.
+
+<h4>Format</h4>
+
+```
+tab
+```
+
 ### Saving data
-Data (all stocks in inventory in json) is automatically saved to
-`[root directory]/data/stockbook.json` when any of these commands is executed:
+Data (all stocks in inventory in JSON) is automatically saved to
+`[root directory]/data/stockbook.JSON` when any of these commands are executed:
```

Comment:
I think the `JSON` on `stockbook.json` should be lowercased as in this context it is a file extension. Same issue with other `JSON` below.

Reply:
Fixed!